packages/svelte/tests/migrate/samples/event-handlers/output.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+	let { onclick, ontoggle, 'custom-event-bubble': oncustom_event_bubble } = $props();
+</script>
+
+<button  onclick={(payload) => {
```

Comment:
maybe `event` rather than `payload`? any idea where the extra space is coming from?

Reply:
it's because there were multiple on:click handlers and their whitespaces where not removed. I think it's rare enough that this situations even happens to not care (and let prettier handle it; I like your suggestion of running prettier on the result within svelte-migrate)